changelogs/fragments/516-whitespace-exos-config-defaults.yaml

```diff
@@ -0,0 +1,3 @@
+bugfixes: >-
+  - exos_config - Missing whitespace in command with ``defaults: True``.
+    It happened because the command is ``show configurationdetail`` instead of ``show configuration detail`` (https://github.com/ansible-collections/community.network/pull/516).
```

Comment:
```suggestion
bugfixes:
  - "exos_config - missing whitespace in command with ``defaults: True``. It happened because the command is ``show configurationdetail`` instead of ``show configuration detail`` (https://github.com/ansible-collections/community.network/pull/516)."
```

Reply:
this will fix the sanity failings ^, could you please commit the change?